internal/pkg/cli/deploy/static_site.go

```diff
@@ -43,16 +41,14 @@ func NewStaticSiteDeployer(in *WorkloadDeployerInput) (*staticSiteDeployer, erro
 	if !ok {
 		return nil, fmt.Errorf("manifest is not of type %s", manifestinfo.StaticSiteType)
 	}
-	assetMappingPath := fmt.Sprintf("local-assets/environment/%s/workloads/%s/mapping/%s.json", svcDeployer.env.Name, svcDeployer.name, time.Now().Format(time.RFC3339))
 	return &staticSiteDeployer{
-		svcDeployer:      svcDeployer,
-		staticSiteMft:    mft,
-		fs:               svcDeployer.fs,
-		assetMappingPath: assetMappingPath,
+		svcDeployer:   svcDeployer,
+		staticSiteMft: mft,
+		fs:            svcDeployer.fs,
 		uploader: &asset.ArtifactBucketUploader{
-			FS:               svcDeployer.fs,
-			PathPrefix:       "local-assets",
-			AssetMappingPath: assetMappingPath,
+			FS:                  svcDeployer.fs,
+			AssetDir:            "local-assets",
```

Comment:
Should we make "local-assets" a constant so that it can be reused for `AssetMappingFileDir`?